### PR TITLE
Add a PEP 541 issue template

### PR DESCRIPTION
**Pull request by yeraydiazdiaz**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pep541-request.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+PEP 541 https://www.python.org/dev/peps/pep-0541/ describes the workflow
+for any project claiming issues and conflict resolution.
+
+Make sure you read and understand the PEP to set your expectations on
+the possible outcomes.
+-->
+
+**Project to be claimed**
+<!--
+Provide the name of the package and ideally a link to its
```
Comment (di) [inline .github/ISSUE_TEMPLATE/pep541-request.md]:
```suggestion
Provide the name of the package and a link to its
```

**Comment by di** [comment]
Thoughts about making these issues go to another repo entirely? The PEP 541 requests tend to pollute the issues of this repo and it's not super intuitive to users that this is the right place.

Also https://github.com/python/peps/pull/1059 hasn't been merged yet, so we could still update it there as well.

Not totally sure what we'd call it, something like pypa/pep541 would maybe be too obscure.

**Comment by pradyunsg** [comment]
> Thoughts about making these issues go to another repo entirely?

I'm in.

> pypa/pep541

Maybe `python/` might be a better place to put it?

**Comment by yeraydiazdiaz** [comment]
I'm 👍 on moving to a separate repo.

I also agree naming the repo as the PEP is confusing though I'm leaning towards keeping in the PyPA org.

Some ideas for naming:

- pypa/transfers
- pypa/project-transfers
- pypa/project-names
- pypa/package-name-retention
- pypa/ownership
- pypa/project-ownership
- pypa/index-names
- pypa/pypi-project-conflict
- pypa/project-conflict-resolution

That last one seems pretty self explanatory?

**Comment by di** [comment]
How about `pypa/pypi-support`? That way we could also move basically any PyPI support request (e.g. account recover) out of these issues and reduce the volume for people that just want to keep an eye on PyPI development.

**Comment by ewdurbin** [comment]
+1 to a generic PyPI support tracker repo.

**Comment by di** [comment]
I've included this at [`pypa/pypi-support`](https://github.com/pypa/pypi-support).

**Pull request closed without merging**